sunpy/data/__init__.py

```diff
@@ -1,3 +1,44 @@
+import astropy.units as u
+
+import sunpy
+from sunpy import config
 from sunpy.data._sample import download_sample_data
+from sunpy.data.data_manager.cache import Cache
+from sunpy.data.data_manager.downloader import ParfiveDownloader
+from sunpy.data.data_manager.manager import DataManager
+from sunpy.data.data_manager.storage import SqliteStorage
+from sunpy.util.config import CACHE_DIR
+
+_download_dir = config.get('downloads', 'remote_data_manager_dir')
+
+
+if hasattr(sunpy, '_called_from_test'):
```

Comment:
Could you put this block inside `pytest_configure` rather than here?

Reply:
I could not get it to work there. I'm probably missing something obvious.